packages/builder/src/pages/builder/app/[application]/_components/BuilderSidePanel.svelte

```diff
@@ -741,16 +744,33 @@
                 <div class="auth-entity-access-title">Access</div>
               </div>
               {#each allUsers as user}
+                {@const userGroups = sdk.users.getUserAppGroups(
+                  $appStore.appId,
+                  user,
+                  $groups
+                )}
                 <div class="auth-entity">
                   <div class="details">
-                    <div class="user-email" title={user.email}>
-                      {user.email}
+                    <div class="user-groups">
+                      <div class="user-email" title={user.email}>
+                        {user.email}
+                      </div>
+                      {#if userGroups.length}
+                        <div class="group-info">
```

Comment:
This takes up a lot of space alongside the email With longer emails it pushes the UI completely out of site.  It may work better to move the group count under the user email or just remove the '2 groups' and leave the icon?

![Screenshot 2025-02-27 at 09 07 54](https://github.com/user-attachments/assets/cf5e0105-cd32-4f50-a7f6-e314344a5274)

Reply:
Thanks for pointing this out, I've gone with truncating long email addresses as that is what would happen if the email was very long already.
![image](https://github.com/user-attachments/assets/1a656da9-ab4e-497c-a5b4-1c26adb60d31)